src/cli/mod.rs

```diff
@@ -25,13 +25,14 @@ pub use self::upload::UploadCommand;
 
 /// Command line options that Rojo accepts, defined using the clap crate.
 #[derive(Debug, Parser)]
-#[clap(name = "Rojo", version, about, author)]
+#[command(name = "Rojo", version, about, author)]
```

Comment:
```suggestion
#[command(name = "Rojo", version, about)]
```

As far as I know, we don't need `author` here if we aren't displaying it to begin with.